data/mods/Magiclysm/achievements/spells.json

```
@@ -0,0 +1,84 @@
+[
+  {
+    "id": "achievement_learn_any_spell",
+    "type": "achievement",
+    "name": "Would-be Wizard",
```

[Other mods](https://github.com/Axema/Secronom-Zombies/blob/master/secronom/Modification%20Files/secro_achievements.json) have been putting the mod name in the achievement name.  Amongst other things, this makes them all sort together in the list, which is nice.  You might want to do the same.
```suggestion
    "name": "<color_light_red>(Magiclysm)</color> Would-be Wizard",
```

---

I had been wondering if there was a category way to sort achievements, though if there were i wouldn't just make a magiclysm category